hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/SummedSharedCounter.java

```diff
@@ -0,0 +1,52 @@
+package pl.allegro.tech.hermes.management.infrastructure.metrics;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.recipes.atomic.DistributedAtomicLong;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import pl.allegro.tech.hermes.infrastructure.zookeeper.counter.ZookeeperCounterException;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.stream.Collectors.toList;
+
+public class SummedSharedCounter {
```

Comment:
What do you think about making this class `final` and `package-scope`?

Reply:
I wanted make this class `package-scope` but this would require injecting `ZookeeperClientManager` to `HybridSubscriptionMetricsRepository` and `HybridTopicMetricsRepository`. I won't do it.

Reply:
`final` keyword would make impossible to create mocks and stubs that we use in `HybridSubscriptionMetricsRepository `.